x-pack/plugins/lens/public/types.ts

```diff
@@ -525,20 +525,10 @@ export interface FramePublicAPI {
    * If accessing, make sure to check whether expected columns actually exist.
    */
   activeData?: Record<string, Datatable>;
-
   dateRange: DateRange;
   query: Query;
   filters: Filter[];
   searchSessionId: string;
-
-  /**
-   * A map of all available palettes (keys being the ids).
-   */
-  availablePalettes: PaletteRegistry;
```

Comment:
That's because they are never used. They are being loaded again in the particular visualizations that need them, for example here: 
https://github.com/elastic/kibana/blob/master/x-pack/plugins/lens/public/datatable_visualization/index.ts#L38